src/website/www/wiki.css

```
@@ -0,0 +1,76 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    display: grid;
+    grid-template-rows: 4rem 100%;
+    grid-template-columns: 25rem auto;
+
+    margin-top: 2rem;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)